schematics/page/index.ts

```diff
@@ -163,6 +163,7 @@ export default function(options: PageOptions): Rule {
 
     const templateSource = apply(url('./files'), [
       options.spec ? noop() : filter(p => !p.endsWith('.spec.ts')),
+      options.createModule ? noop() : filter(p => !p.endsWith('.module.ts')),
```

Comment:
It would be nice if we could add some unit tests to validate these changes and keep align with existing behavior. :)